app.py

```diff
@@ -130,12 +129,10 @@ def inject_global_constants():
 
     @flask_app.context_processor
     def inject_service_name():
-        try:
-            fund_title = FundMethods.get_service_name()
+        fund_title = FundMethods.get_service_name()
+        if fund_title:
             return dict(service_title=gettext("Apply for") + " " + fund_title)
-
-        except Exception as e:  # noqa
-            current_app.log_exception(e)
+        else:
             return dict(service_title="Access Funding")
```

Comment:
Think this could be simplified into one ternary statement? e.g.
```
return dict(service_title=gettext("Apply for") + " " + fund_title) if fund_title else dict(service_title="Access Funding")
```

Reply:
have changed as suggested! 💯